packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1261,6 +1263,18 @@ class RenderBackdropFilter extends RenderProxyBox {
     _blendMode = value;
     markNeedsPaint();
   }
+  /// The backdrop key that identifies the backdrop this filter will apply to.
+  ///
+  /// The default value for the backdrop key is `null`.
+  BackdropKey? get backdropKey => _backdropKey;
+  BackdropKey? _backdropKey;
+  set backdropKey(BackdropKey? value) {
```

Comment:
Do you think this needs to be modifiable?

Reply:
wdym?

Reply:
This field can be specified in the constructor, do we need to edit it after it's constructed?  It seems like specifying it in the constructor is sufficient.

Reply:
Oh, the setter. in general yeah since the render box can be recycled as the widgets change around.